web/src/routes/RepositoriesList/RepositoriesList.tsx

```diff
@@ -414,3 +429,7 @@ interface RepoListTableItem {
   size: number;
   last_modified: number;
 }
+
+interface RepositoriesListProps {
+  organizationName: string;
```

Comment:
Is there a reason for adding the props here? I don't see the `organizationName` getting used in the PR.

Reply:
I see `RepositoriesListProps` used in `src/routes/RepositoriesList/RepositoriesList.tsx:65` but it's not defined so I'm defining it